### DISCUSSION TITLE
Reporting configuration file

### DISCUSSION THREAD

**Discussion by Mebrahem1** [projectdiscovery/nuclei#1397] [Q&A] [answered]
Hi everyone,

I need help,

I used `-rc, -report-config` before, and it worked fine without problems, one day I reinstalled my ubuntu 20.04 OS on my VPS server, and when I downloaded the latest version of nuclei I tried to use `-rc, -report-config ` But it never worked I tried a lot but it didn't work, I thought there was a problem with my token in my github account so I used `curl` to make sure that the token works or not and I found it works fine without any problems, I tried to use gitlab and found that it doesn't work either!

**My command:**
```
nuclei -u http://site.com/etc -severity medium -headless -sr -ev -v -t /root/nuclei-templates/cves/ -rc /PATH/to/report-config.yaml
```

Thanks.

**Comment by Mebrahem1** [comment]
any help?

**Comment by Mebrahem1** [comment, marked as answer]
Solved!

I found this https://github.com/projectdiscovery/nuclei/blob/master/v2/cmd/nuclei/issue-tracker-config.yaml not working, but this https://github.com/projectdiscovery/nuclei/blob/e0e48837c6ff649cbe954600e6c643f5b82db96f/integration_tests/test-issue-tracker-config1.yaml is working fine!

**Reply by ehsandeep** [reply]
thanks for pointing it out @Mebrahem1, I've updated the example issue tracker here - https://github.com/projectdiscovery/nuclei/commit/8e62abe6e7f42fec1b49da727c1e5e5746e08cf3
